src/api/lib/rabbitmq_bus.rb

```diff
@@ -2,7 +2,7 @@ class RabbitmqBus
   cattr_accessor :connection, :exchange, :failed
 
   def self.send_to_bus(channel, data)
-    channel = "#{Configuration.amqp_namespace}.#{channel}"
+    channel = "#{CONFIG['amqp_namespace']}.#{channel}"
```

Comment:
That's because there are 3 SQL queries are the result of calling `first` 3 times, inside method_missing. However the method `amqp_namespace` exists. We should fix that. 

The method:

```
  def amqp_namespace
    CONFIG['amqp_namespace'] || 'opensuse.obs'
  end
```
shouldn't generate any SQL query other than calling `CONFIG['amqp_namespace']`

Reply:
@eduardoj suggested to go for [`config_for`](https://guides.rubyonrails.org/configuring.html#custom-configuration) instead of all of this custom code. I dropped the commit.